devicon.json

```
@@ -4753,6 +4753,27 @@
         "color": "#3d3b47",
         "aliases": []
     },
+    {
+        "name": "norg",
+        "altnames": [
+            "neorg"
+        ],
+        "tags": [
+            "note-taking",
+            "organization"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#4793af",
```

I can't find this color in any of the icons, did you perhaps forget to change it?
```suggestion
        "color": "#366794",
```

Couldn't find any official color for their branding, but imo one of the most prevalent color from the logo will do. The one suggested above is just a suggestion. You can pick one of the other colors in the logo if you want, or even better, an official branding color if you can find it

---

We actually got this color by getting an average of the colors in the official logo, but the one you have provided looks much better! Thanks again :)

---

Ahh, that makes sense. You decide which one you choose then :)

---

So after a bit of deciding we've decided to go with `#4878be` which is a bit of a lighter color from the one you suggested